text-ui-test/input.txt

```diff
@@ -0,0 +1 @@
+todo borrow book
```

Comment:
test input could be more comprehensive, perhaps at least touch on each of your main features :D

Reply:
At this moment in time I did not manage to push my test inputs with deadline and events unfortunately, but thank you for the comment